### PR TITLE
[zh-cn] sync tasks/administer-cluster/*

### PR DESCRIPTION
**Pull request by my-git9**
```
# Changes to be committed: 
#       modified:   content/zh-cn/docs/tasks/administer-cluster/declare-network-policy.md 
#       modified:   content/zh-cn/docs/tasks/administer-cluster/dns-debugging-resolution.md 
#       modified:   content/zh-cn/docs/tasks/administer-cluster/dns-horizontal-autoscaling.md 
#       modified:   content/zh-cn/docs/tasks/administer-cluster/manage-resources/quota-memory-cpu-namespace.md 
#       modified:   content/zh-cn/docs/tasks/administer-cluster/manage-resources/quota-pod-namespace.md 
#       modified:   content/zh-cn/docs/tasks/administer-cluster/namespaces-walkthrough.md 
#       modified:   content/zh-cn/docs/tasks/administer-cluster/quota-api-object.md 
#       modified:   content/zh-cn/docs/tasks/administer-cluster/running-cloud-controller.md 
#       modified:   content/zh-cn/docs/tasks/administer-cluster/verify-signed-artifacts.md 

```

### CODE REVIEW

File: content/zh-cn/docs/tasks/administer-cluster/verify-signed-artifacts.md [hunk 1]
```diff
@@ -182,3 +182,49 @@ Here are some helpful resources to get started with `policy-controller`:
...
+# Retrieve sigstore signature and certificate
+
+# Verify the sigstore signature
+-->
+
+```shell
+# 检索最新可用的 Kubernetes 发行版本
+VERSION=$(curl -Ls https://dl.k8s.io/release/stable.txt)
+
+# 验证 SHA512 sum
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/verify-signed-artifacts.md:212]: 下同
```suggestion
# 检查 SHA512 校验和
```

**Review by windsonsea — COMMENTED** [review]
/lgtm

**Pull request merged**